ui/shared/status/StatusButton.qml

```diff
@@ -8,6 +8,7 @@ Button {
     property string type: "primary"
     property string size: "large"
     property string state: "default"
+    property string color: Style.current.buttonForegroundColor
```

Comment:
The color type is pretty similar to string, but in QT creator, it shows a color picker so it's helpful
```suggestion
    property color color: Style.current.buttonForegroundColor
```